2015-04-27 01:10:42.535 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 01:12:22.416 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 01:45:35.373 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 01:49:15.044 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 01:49:35.173 [warning] <0.338.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-27 02:17:47.399 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 02:18:12.395 [warning] <0.338.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-27 02:18:36.021 [error] <0.447.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.566.0> exit with reason: {[{reason,badarg},{mfa,{ibot_webi_handler,websocket_handle,3}},{stacktrace,[{erlang,list_to_atom,"B",[]},{ibot_nodes_srv_connector,stop_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,136}]},{ibot_webi_handler,websocket_handle,3,[{file,"src/ibot_webi_handler.erl"},{line,81}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]},{msg,{text,<<"{\"stopNode\" : \"BLA_BLA_BLA_PUB\"}">>}},{req,[{socket,#Port<0.796>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.566.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},57765}},{host,<<"localhost">>},{host_info,undefined},{port,10100},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10100">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10100">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"bald9ZlsG09/eHFJsZ2Vag==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}],[{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]}

2015-04-27 02:18:36.021 [error] emulator Error in process <0.566.0> on node 'core@alex-N550JK' with exit value: {[{reason,badarg},{mfa,{ibot_webi_handler,websocket_handle,3}},{stacktrace,[{erlang,list_to_atom,"B",[]},{ibot_nodes_srv_connector,stop_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,136}]},{ibot_webi_handler,websocket_handle... 


2015-04-27 02:19:12.416 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 02:20:55.344 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 02:21:14.390 [warning] <0.338.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-27 02:21:32.753 [error] <0.447.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.566.0> exit with reason: {[{reason,badarg},{mfa,{ibot_webi_handler,websocket_handle,3}},{stacktrace,[{erlang,list_to_atom,"B",[]},{ibot_nodes_srv_connector,stop_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,137}]},{ibot_webi_handler,websocket_handle,3,[{file,"src/ibot_webi_handler.erl"},{line,81}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]},{msg,{text,<<"{\"stopNode\" : \"BLA_BLA_BLA_PUB\"}">>}},{req,[{socket,#Port<0.796>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.566.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},57784}},{host,<<"localhost">>},{host_info,undefined},{port,10100},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10100">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10100">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"6ViYUBeBQvwtvDFDhESlXQ==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}],[{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]}

2015-04-27 02:21:32.753 [error] emulator Error in process <0.566.0> on node 'core@alex-N550JK' with exit value: {[{reason,badarg},{mfa,{ibot_webi_handler,websocket_handle,3}},{stacktrace,[{erlang,list_to_atom,"B",[]},{ibot_nodes_srv_connector,stop_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,137}]},{ibot_webi_handler,websocket_handle... 


2015-04-27 02:21:42.676 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 02:21:52.895 [warning] <0.338.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-27 02:21:52.895 [error] emulator Error in process <0.567.0> on node 'core@alex-N550JK' with exit value: {[{reason,badarg},{mfa,{ibot_webi_handler,websocket_handle,3}},{stacktrace,[{erlang,list_to_atom,"B",[]},{ibot_nodes_srv_connector,stop_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,137}]},{ibot_webi_handler,websocket_handle... 


2015-04-27 02:21:52.897 [error] <0.447.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.567.0> exit with reason: {[{reason,badarg},{mfa,{ibot_webi_handler,websocket_handle,3}},{stacktrace,[{erlang,list_to_atom,"B",[]},{ibot_nodes_srv_connector,stop_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,137}]},{ibot_webi_handler,websocket_handle,3,[{file,"src/ibot_webi_handler.erl"},{line,81}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]},{msg,{text,<<"{\"stopNode\" : \"BLA_BLA_BLA_PUB\"}">>}},{req,[{socket,#Port<0.794>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.567.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},57797}},{host,<<"localhost">>},{host_info,undefined},{port,10100},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10100">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10100">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"XIX7bY6xSHMw0e43Ou0+dQ==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}],[{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]}

2015-04-27 02:22:25.049 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 02:24:15.358 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 02:24:25.332 [warning] <0.338.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-27 02:27:24.759 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 02:27:57.751 [warning] <0.338.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-27 02:36:21.765 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 02:37:02.795 [warning] <0.338.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-27 02:39:56.965 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 02:40:09.681 [warning] <0.338.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-27 02:45:22.063 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 02:45:35.388 [warning] <0.338.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-27 02:46:39.468 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 02:46:49.715 [warning] <0.338.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-27 02:48:13.780 [error] <0.574.0> gen_server bar_monitor terminated with reason: {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in gen_server:call/2 line 182
2015-04-27 02:48:14.079 [error] <0.574.0> CRASH REPORT Process bar_monitor with 1 neighbours exited with reason: {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in gen_server:terminate/7 line 804
2015-04-27 02:48:14.080 [error] <0.553.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link([{node_info,"ClientTest",undefined,undefined,undefined,"alex-N550JK",undefined,"bar@alex-N550JK",...},...]) at <0.555.0> exit with reason {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in context child_terminated
2015-04-27 02:49:48.765 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 02:50:09.039 [warning] <0.338.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-27 02:51:26.044 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 02:52:15.442 [warning] <0.338.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-27 02:53:49.542 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 02:55:12.305 [warning] <0.338.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-27 21:58:50.008 [info] <0.339.0> Application lager started on node 'core@alex-N550JK'
2015-04-27 21:59:17.244 [warning] <0.338.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-29 01:10:35.573 [info] <0.368.0> Application lager started on node 'core@alex-N550JK'
2015-04-29 01:29:07.170 [warning] <0.367.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-29 01:29:07.171 [error] <0.601.0> gen_server ibot_nodes_srv_connector terminated with reason: no function clause matching ibot_nodes_srv_connector:run_node({atomic,[{node_info,'BLA_BLA_BLA',"BLA_BLA_BLA","BLA_BLA_BLA_MBoxAsync",'BLA_BLA_BLA_MBoxAsync',...}]}) line 96
2015-04-29 01:29:07.171 [error] <0.601.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: no function clause matching ibot_nodes_srv_connector:run_node({atomic,[{node_info,'BLA_BLA_BLA',"BLA_BLA_BLA","BLA_BLA_BLA_MBoxAsync",'BLA_BLA_BLA_MBoxAsync',...}]}) line 96 in gen_server:terminate/7 line 804
2015-04-29 01:29:07.171 [error] <0.599.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link([{node_info,undefined,"ClientTest",undefined,undefined,"alex-N550JK",undefined,"bar@alex-N550JK",...},...]) at <0.601.0> exit with reason no function clause matching ibot_nodes_srv_connector:run_node({atomic,[{node_info,'BLA_BLA_BLA',"BLA_BLA_BLA","BLA_BLA_BLA_MBoxAsync",'BLA_BLA_BLA_MBoxAsync',...}]}) line 96 in context child_terminated
2015-04-29 01:33:18.980 [info] <0.368.0> Application lager started on node 'core@alex-N550JK'
2015-04-29 01:33:34.897 [warning] <0.367.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 01:54:15.829 [info] <0.368.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 01:54:40.784 [warning] <0.367.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 01:54:40.785 [error] <0.437.0> gen_server ibot_db_srv terminated with reason: bad argument in call to ets:delete(ibot_node_info_table) in ibot_db_func:delete_table/1 line 27
2015-04-30 01:54:40.785 [error] <0.437.0> CRASH REPORT Process ibot_db_srv with 0 neighbours exited with reason: bad argument in call to ets:delete(ibot_node_info_table) in ibot_db_func:delete_table/1 line 27
2015-04-30 01:54:40.785 [error] <0.436.0> Supervisor ibot_db_sup had child ibot_db_srv started with ibot_db_srv:start_link() at <0.437.0> exit with reason bad argument in call to ets:delete(ibot_node_info_table) in ibot_db_func:delete_table/1 line 27 in context child_terminated
2015-04-30 01:54:40.785 [error] <0.432.0> gen_server ibot_core_srv_compile_nodes terminated with reason: bad argument in call to ets:delete(ibot_node_info_table) in ibot_db_func:delete_table/1 line 27 in gen_server:call/2 line 182
2015-04-30 01:54:40.786 [error] <0.432.0> CRASH REPORT Process ibot_core_srv_compile_nodes with 0 neighbours exited with reason: bad argument in call to ets:delete(ibot_node_info_table) in ibot_db_func:delete_table/1 line 27 in gen_server:terminate/7 line 804
2015-04-30 01:54:40.786 [error] <0.430.0> Supervisor ibot_core_sup had child ibot_core_srv_compile_nodes started with ibot_core_srv_compile_nodes:start_link() at <0.432.0> exit with reason bad argument in call to ets:delete(ibot_node_info_table) in ibot_db_func:delete_table/1 line 27 in context child_terminated
2015-04-30 01:54:40.787 [error] <0.494.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.611.0> exit with reason: [{reason,{{{badarg,[{ets,delete,[ibot_node_info_table],[]},{ibot_db_func,delete_table,1,[{file,"src/ibot_db_func.erl"},{line,27}]},{ibot_db_srv,terminate,2,[{file,"src/ibot_db_srv.erl"},{line,132}]},{gen_server,try_terminate,3,[{file,"gen_server.erl"},{line,621}]},{gen_server,terminate,7,[{file,"gen_server.erl"},{line,787}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[ibot_db_srv,{get_record,ibot_node_info_table,'BLA_BLA_BLA'}]}},{gen_server,call,[ibot_core_srv_compile_nodes,{compile_node,"BLA_BLA_BLA"}]}}},{mfa,{ibot_webi_handler,websocket_handle,3}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{ibot_webi_handler,websocket_handle,3,[{file,"src/ibot_webi_handler.erl"},{line,42}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]},{msg,{text,<<"{\"compileOneNode\" : \"BLA_BLA_BLA\"}">>}},{req,[{socket,#Port<0.857>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.611.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},51791}},{host,<<"localhost">>},{host_info,undefined},{port,10100},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10100">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10100">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"IHEaESn2pTPfPszOKTqxZw==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}]

2015-04-30 01:56:00.889 [info] <0.368.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 01:57:23.649 [warning] <0.367.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 01:57:23.649 [info] <0.368.0> Application mnesia exited with reason: stopped
2015-04-30 01:57:23.649 [error] <0.437.0> gen_server ibot_db_srv terminated with reason: bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24
2015-04-30 01:57:23.649 [error] <0.437.0> CRASH REPORT Process ibot_db_srv with 0 neighbours exited with reason: bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24
2015-04-30 01:57:23.649 [error] <0.436.0> Supervisor ibot_db_sup had child ibot_db_srv started with ibot_db_srv:start_link() at <0.437.0> exit with reason bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24 in context child_terminated
2015-04-30 01:57:23.649 [error] <0.432.0> gen_server ibot_core_srv_compile_nodes terminated with reason: bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24 in gen_server:call/2 line 182
2015-04-30 01:57:23.649 [error] <0.432.0> CRASH REPORT Process ibot_core_srv_compile_nodes with 0 neighbours exited with reason: bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24 in gen_server:terminate/7 line 804
2015-04-30 01:57:23.650 [error] <0.430.0> Supervisor ibot_core_sup had child ibot_core_srv_compile_nodes started with ibot_core_srv_compile_nodes:start_link() at <0.432.0> exit with reason bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24 in context child_terminated
2015-04-30 01:57:23.650 [error] <0.494.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.612.0> exit with reason: [{reason,{{{badarg,[{ets,lookup,[ibot_node_info_table,'BLA_BLA_BLA'],[]},{ibot_db_func,get,2,[{file,"src/ibot_db_func.erl"},{line,24}]},{ibot_db_srv,handle_call,3,[{file,"src/ibot_db_srv.erl"},{line,89}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[ibot_db_srv,{get_record,ibot_node_info_table,'BLA_BLA_BLA'}]}},{gen_server,call,[ibot_core_srv_compile_nodes,{compile_node,"BLA_BLA_BLA"}]}}},{mfa,{ibot_webi_handler,websocket_handle,3}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{ibot_webi_handler,websocket_handle,3,[{file,"src/ibot_webi_handler.erl"},{line,42}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]},{msg,{text,<<"{\"compileOneNode\" : \"BLA_BLA_BLA\"}">>}},{req,[{socket,#Port<0.858>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.612.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},51796}},{host,<<"localhost">>},{host_info,undefined},{port,10100},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10100">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10100">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"idwaz1O2XNENaTI5hmAwLw==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}]

2015-04-30 01:57:23.680 [info] <0.368.0> Application mnesia started on node 'core@alex-N550JK'
2015-04-30 02:02:39.277 [info] <0.368.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 02:04:47.624 [info] <0.368.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 02:04:55.126 [warning] <0.367.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 02:04:55.126 [info] <0.368.0> Application mnesia exited with reason: stopped
2015-04-30 02:04:55.126 [error] <0.437.0> gen_server ibot_db_srv terminated with reason: bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24
2015-04-30 02:04:55.127 [error] <0.437.0> CRASH REPORT Process ibot_db_srv with 0 neighbours exited with reason: bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24
2015-04-30 02:04:55.127 [error] <0.436.0> Supervisor ibot_db_sup had child ibot_db_srv started with ibot_db_srv:start_link() at <0.437.0> exit with reason bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24 in context child_terminated
2015-04-30 02:04:55.127 [error] <0.432.0> gen_server ibot_core_srv_compile_nodes terminated with reason: bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24 in gen_server:call/2 line 182
2015-04-30 02:04:55.127 [error] <0.432.0> CRASH REPORT Process ibot_core_srv_compile_nodes with 0 neighbours exited with reason: bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24 in gen_server:terminate/7 line 804
2015-04-30 02:04:55.128 [error] <0.430.0> Supervisor ibot_core_sup had child ibot_core_srv_compile_nodes started with ibot_core_srv_compile_nodes:start_link() at <0.432.0> exit with reason bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24 in context child_terminated
2015-04-30 02:04:55.129 [error] <0.494.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.612.0> exit with reason: [{reason,{{{badarg,[{ets,lookup,[ibot_node_info_table,'BLA_BLA_BLA'],[]},{ibot_db_func,get,2,[{file,"src/ibot_db_func.erl"},{line,24}]},{ibot_db_srv,handle_call,3,[{file,"src/ibot_db_srv.erl"},{line,89}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[ibot_db_srv,{get_record,ibot_node_info_table,'BLA_BLA_BLA'}]}},{gen_server,call,[ibot_core_srv_compile_nodes,{compile_node,"BLA_BLA_BLA"}]}}},{mfa,{ibot_webi_handler,websocket_handle,3}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{ibot_webi_handler,websocket_handle,3,[{file,"src/ibot_webi_handler.erl"},{line,42}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]},{msg,{text,<<"{\"compileOneNode\" : \"BLA_BLA_BLA\"}">>}},{req,[{socket,#Port<0.858>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.612.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},51852}},{host,<<"localhost">>},{host_info,undefined},{port,10100},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10100">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10100">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"Tdve6LSIex4NpVw2NajBzw==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}]

2015-04-30 02:04:55.171 [info] <0.368.0> Application mnesia started on node 'core@alex-N550JK'
2015-04-30 02:06:15.916 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 02:06:24.673 [warning] <0.364.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 02:06:24.673 [info] <0.365.0> Application mnesia exited with reason: stopped
2015-04-30 02:06:24.673 [error] <0.434.0> gen_server ibot_db_srv terminated with reason: bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24
2015-04-30 02:06:24.673 [error] <0.434.0> CRASH REPORT Process ibot_db_srv with 0 neighbours exited with reason: bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24
2015-04-30 02:06:24.673 [error] <0.433.0> Supervisor ibot_db_sup had child ibot_db_srv started with ibot_db_srv:start_link() at <0.434.0> exit with reason bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24 in context child_terminated
2015-04-30 02:06:24.673 [error] <0.429.0> gen_server ibot_core_srv_compile_nodes terminated with reason: bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24 in gen_server:call/2 line 182
2015-04-30 02:06:24.673 [error] <0.429.0> CRASH REPORT Process ibot_core_srv_compile_nodes with 0 neighbours exited with reason: bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24 in gen_server:terminate/7 line 804
2015-04-30 02:06:24.673 [error] <0.427.0> Supervisor ibot_core_sup had child ibot_core_srv_compile_nodes started with ibot_core_srv_compile_nodes:start_link() at <0.429.0> exit with reason bad argument in call to ets:lookup(ibot_node_info_table, 'BLA_BLA_BLA') in ibot_db_func:get/2 line 24 in context child_terminated
2015-04-30 02:06:24.674 [error] <0.491.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.610.0> exit with reason: [{reason,{{{badarg,[{ets,lookup,[ibot_node_info_table,'BLA_BLA_BLA'],[]},{ibot_db_func,get,2,[{file,"src/ibot_db_func.erl"},{line,24}]},{ibot_db_srv,handle_call,3,[{file,"src/ibot_db_srv.erl"},{line,89}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[ibot_db_srv,{get_record,ibot_node_info_table,'BLA_BLA_BLA'}]}},{gen_server,call,[ibot_core_srv_compile_nodes,{compile_node,"BLA_BLA_BLA"}]}}},{mfa,{ibot_webi_handler,websocket_handle,3}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{ibot_webi_handler,websocket_handle,3,[{file,"src/ibot_webi_handler.erl"},{line,42}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]},{msg,{text,<<"{\"compileOneNode\" : \"BLA_BLA_BLA\"}">>}},{req,[{socket,#Port<0.853>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.610.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},51891}},{host,<<"localhost">>},{host_info,undefined},{port,10100},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10100">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10100">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"eQFu85rhWXr01Y89UKGH+Q==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}]

2015-04-30 02:06:24.724 [info] <0.365.0> Application mnesia started on node 'core@alex-N550JK'
2015-04-30 02:07:51.993 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 02:08:41.631 [warning] <0.364.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 02:11:39.798 [error] <0.600.0> gen_server ibot_nodes_srv_service terminated with reason: no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61
2015-04-30 02:11:39.798 [error] <0.600.0> CRASH REPORT Process ibot_nodes_srv_service with 0 neighbours exited with reason: no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61 in gen_server:terminate/7 line 804
2015-04-30 02:11:39.798 [error] <0.597.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_service started with ibot_nodes_srv_service:start_link() at <0.600.0> exit with reason no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61 in context child_terminated
2015-04-30 02:11:39.798 [error] <0.630.0> gen_server ibot_nodes_srv_service terminated with reason: no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61
2015-04-30 02:11:39.798 [error] <0.630.0> CRASH REPORT Process ibot_nodes_srv_service with 0 neighbours exited with reason: no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61 in gen_server:terminate/7 line 804
2015-04-30 02:11:39.799 [error] <0.597.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_service started with ibot_nodes_srv_service:start_link() at <0.630.0> exit with reason no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61 in context child_terminated
2015-04-30 02:11:39.799 [error] <0.631.0> gen_server ibot_nodes_srv_service terminated with reason: no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61
2015-04-30 02:11:39.799 [error] <0.631.0> CRASH REPORT Process ibot_nodes_srv_service with 0 neighbours exited with reason: no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61 in gen_server:terminate/7 line 804
2015-04-30 02:11:39.799 [error] <0.597.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_service started with ibot_nodes_srv_service:start_link() at <0.631.0> exit with reason no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61 in context child_terminated
2015-04-30 02:11:39.799 [error] <0.632.0> gen_server ibot_nodes_srv_service terminated with reason: no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61
2015-04-30 02:11:39.799 [error] <0.632.0> CRASH REPORT Process ibot_nodes_srv_service with 0 neighbours exited with reason: no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61 in gen_server:terminate/7 line 804
2015-04-30 02:11:39.799 [error] <0.597.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_service started with ibot_nodes_srv_service:start_link() at <0.632.0> exit with reason no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61 in context child_terminated
2015-04-30 02:11:39.800 [error] <0.633.0> gen_server ibot_nodes_srv_service terminated with reason: no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61
2015-04-30 02:11:39.800 [error] <0.633.0> CRASH REPORT Process ibot_nodes_srv_service with 0 neighbours exited with reason: no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61 in gen_server:terminate/7 line 804
2015-04-30 02:11:39.800 [error] <0.597.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_service started with ibot_nodes_srv_service:start_link() at <0.633.0> exit with reason no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61 in context child_terminated
2015-04-30 02:11:39.800 [error] <0.634.0> gen_server ibot_nodes_srv_service terminated with reason: no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61
2015-04-30 02:11:39.800 [error] <0.634.0> CRASH REPORT Process ibot_nodes_srv_service with 0 neighbours exited with reason: no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61 in gen_server:terminate/7 line 804
2015-04-30 02:11:39.800 [error] <0.597.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_service started with ibot_nodes_srv_service:start_link() at <0.634.0> exit with reason no case clause matching {aborted,{no_exists,service_server}} in ibot_db_func_services:get_server_service/1 line 61 in context child_terminated
2015-04-30 02:11:39.800 [error] <0.597.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_service started with ibot_nodes_srv_service:start_link() at <0.634.0> exit with reason reached_max_restart_intensity in context shutdown
2015-04-30 02:15:05.846 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 02:17:59.394 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 02:53:29.176 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 02:53:51.429 [warning] <0.364.0> lager_error_logger_h dropped 56 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 02:55:19.177 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 02:55:28.694 [warning] <0.364.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 02:59:53.873 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 03:00:04.229 [warning] <0.364.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 03:03:11.357 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 03:03:25.363 [warning] <0.364.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 03:07:34.153 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 03:08:03.709 [warning] <0.364.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 03:12:00.153 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 03:14:12.298 [warning] <0.364.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 03:14:17.303 [error] <0.599.0> gen_server ibot_nodes_srv_topic terminated with reason: bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 31
2015-04-30 03:14:17.303 [error] <0.599.0> CRASH REPORT Process ibot_nodes_srv_topic with 0 neighbours exited with reason: bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 31 in gen_server:terminate/7 line 804
2015-04-30 03:14:17.303 [error] <0.598.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_topic started with ibot_nodes_srv_topic:start_link() at <0.599.0> exit with reason bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 31 in context child_terminated
2015-04-30 03:15:16.998 [error] <0.618.0> gen_server ibot_nodes_srv_topic terminated with reason: bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 31
2015-04-30 03:15:16.998 [error] <0.618.0> CRASH REPORT Process ibot_nodes_srv_topic with 0 neighbours exited with reason: bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 31 in gen_server:terminate/7 line 804
2015-04-30 03:15:16.998 [error] <0.598.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_topic started with ibot_nodes_srv_topic:start_link() at <0.618.0> exit with reason bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 31 in context child_terminated
2015-04-30 03:19:38.437 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 03:19:55.958 [warning] <0.364.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 03:22:03.270 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 03:22:16.316 [warning] <0.364.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 03:25:49.830 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 03:26:06.898 [warning] <0.364.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 03:26:34.397 [error] emulator Error in process <0.619.0> on node 'core@alex-N550JK' with exit value: {{badrecord,node_pubsub_info},[{ibot_nodes_srv_topic,message_broadcast,3,[{file,"src/ibot_nodes_srv_topic.erl"},{line,133}]}]}


2015-04-30 03:26:39.399 [error] emulator Error in process <0.620.0> on node 'core@alex-N550JK' with exit value: {{badrecord,node_pubsub_info},[{ibot_nodes_srv_topic,message_broadcast,3,[{file,"src/ibot_nodes_srv_topic.erl"},{line,133}]}]}


2015-04-30 03:26:44.398 [error] emulator Error in process <0.621.0> on node 'core@alex-N550JK' with exit value: {{badrecord,node_pubsub_info},[{ibot_nodes_srv_topic,message_broadcast,3,[{file,"src/ibot_nodes_srv_topic.erl"},{line,133}]}]}


2015-04-30 03:26:49.401 [error] emulator Error in process <0.622.0> on node 'core@alex-N550JK' with exit value: {{badrecord,node_pubsub_info},[{ibot_nodes_srv_topic,message_broadcast,3,[{file,"src/ibot_nodes_srv_topic.erl"},{line,133}]}]}


2015-04-30 03:32:19.549 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-04-30 03:32:30.050 [warning] <0.364.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-04-30 03:33:43.947 [error] <0.599.0> gen_server ibot_nodes_srv_topic terminated with reason: bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36
2015-04-30 03:33:43.948 [error] <0.599.0> CRASH REPORT Process ibot_nodes_srv_topic with 0 neighbours exited with reason: bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36 in gen_server:terminate/7 line 804
2015-04-30 03:33:43.948 [error] <0.598.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_topic started with ibot_nodes_srv_topic:start_link() at <0.599.0> exit with reason bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36 in context child_terminated
2015-05-02 23:54:35.567 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-05-02 23:55:06.351 [warning] <0.364.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-05-02 23:55:59.249 [error] <0.599.0> gen_server ibot_nodes_srv_topic terminated with reason: bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36
2015-05-02 23:55:59.249 [error] <0.599.0> CRASH REPORT Process ibot_nodes_srv_topic with 0 neighbours exited with reason: bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36 in gen_server:terminate/7 line 804
2015-05-02 23:55:59.249 [error] <0.598.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_topic started with ibot_nodes_srv_topic:start_link() at <0.599.0> exit with reason bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36 in context child_terminated
2015-05-02 23:57:26.305 [error] <0.629.0> gen_server ibot_nodes_srv_topic terminated with reason: bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36
2015-05-02 23:57:26.305 [error] <0.629.0> CRASH REPORT Process ibot_nodes_srv_topic with 0 neighbours exited with reason: bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36 in gen_server:terminate/7 line 804
2015-05-02 23:57:26.305 [error] <0.598.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_topic started with ibot_nodes_srv_topic:start_link() at <0.629.0> exit with reason bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36 in context child_terminated
2015-05-02 23:57:43.432 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-05-02 23:58:00.146 [warning] <0.364.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-05-02 23:58:15.578 [error] <0.599.0> gen_server ibot_nodes_srv_topic terminated with reason: bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36
2015-05-02 23:58:15.578 [error] <0.599.0> CRASH REPORT Process ibot_nodes_srv_topic with 0 neighbours exited with reason: bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36 in gen_server:terminate/7 line 804
2015-05-02 23:58:15.579 [error] <0.598.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_topic started with ibot_nodes_srv_topic:start_link() at <0.599.0> exit with reason bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36 in context child_terminated
2015-05-03 00:02:21.402 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-05-03 00:02:30.018 [warning] <0.364.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-05-03 00:03:01.341 [error] <0.599.0> gen_server ibot_nodes_srv_topic terminated with reason: bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36
2015-05-03 00:03:01.341 [error] <0.599.0> CRASH REPORT Process ibot_nodes_srv_topic with 0 neighbours exited with reason: bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36 in gen_server:terminate/7 line 804
2015-05-03 00:03:01.342 [error] <0.598.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_topic started with ibot_nodes_srv_topic:start_link() at <0.599.0> exit with reason bad record topic_info in ibot_db_func_topics:add_node_to_topic/3 line 36 in context child_terminated
2015-05-03 00:04:31.107 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-05-03 00:04:42.031 [warning] <0.364.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-05-03 00:07:52.079 [info] <0.365.0> Application lager started on node 'core@alex-N550JK'
2015-05-03 00:08:04.570 [warning] <0.364.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
